libs/ui-toolkit/src/components/notification/notification.tsx

```diff
@@ -40,7 +45,8 @@ export const Notification = ({
           'border-yellow-500': intent === Intent.Warning,
           'border-vega-pink': intent === Intent.Danger,
         },
-        'border rounded px-3 py-1 text-xs mb-1 mr-1'
+        'border rounded-[4px] text-xs mb-1 mr-1 p-4 flex items-start gap-2.5 bg-neutral-100 dark:bg-neutral-900',
```

Comment:
suggestion (non blocking): Can we just use `rounded` here. I'd like to avoid specifying exact pixel values in TW classes. The more we allow it the more we will end up with an inconsistent design later on..

Reply:
sure, done, thanks